File: src/main/java/org/jukeboxmc/item/enchantment/EnchantmentLooting.java
package org.jukeboxmc.item.enchantment;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class EnchantmentLooting extends Enchantment {

    @Override
    public short getId() {
        return 14;
    }

    @Override
    public int getMaxLevel() {
        return 3;
    }

    @Override
    public int getWeight() {
        return 2;
    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/item/enchantment/EnchantmentLooting.kt
package org.jukeboxmc.server.item.enchantment

import org.jukeboxmc.api.item.enchantment.Rarity

class EnchantmentLooting : JukeboxEnchantmentLoot() {

    override fun getId(): Int {
        return 14
    }

    override fun getMaxLevel(): Int {
        return 3
    }

    override fun getRarity(): Rarity {
        return Rarity.RARE
    }

}
